src/test/resources/application-test-security.yml

```
@@ -0,0 +1,70 @@
+clothes:
+  extraction:
+    musinsa:
+      image-url-prefix: asd
+
+jwt:
+  issuer: weatherwear
+  access-token:
+    validity-seconds: 900 # 15분
+  refresh-token:
+    validity-seconds: 2592000 # 30일
+  secret: test-secret-key-must-be-at-least-32-characters-long
```

YAMLlint (1.37.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)